doc/source/cluster/kubernetes/k8s-ecosystem/volcano.md

```diff
@@ -174,8 +174,11 @@ kubectl get queue kuberay-test-queue -o yaml
 #   reclaimable: true
 #   weight: 1
 # status:
+#   allocated:
+#     cpu: "3"
+#     memory: 4Gi 
+#     pods: "3"
```

Comment:
Instead of `running: 1`, it changes to `allocated: ...`. The description above has changed also.
<img width="1450" alt="volcano-queue-output" src="https://github.com/user-attachments/assets/5457f160-f80f-443e-970b-68291d75687d" />